lib/proofer/vendor/mock.rb

```diff
@@ -22,18 +22,18 @@ def submit_answers(question_set, session_id = nil)
 
       def submit_financials(financials, session_id = nil)
         if financials.is_a?(Hash) && financials.values.first == '12345678'
```

Comment:
I have made the "bad" values 555-555-5555 for phone and 00000000 for financials.